channels/mx.m3u

```diff
@@ -69,3 +79,5 @@ http://wms.tecnoxia.com:1935/8006/8006/playlist.m3u
 http://wms.tecnoxia.com:1935/8006/8006/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Spanish" tvg-logo="https://i.imgur.com/bjGgLrA.jpg" group-title="",Telemax
 http://s5.mexside.net:1935/telemax/telemax/.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Spanish" tvg-logo="https://i.imgur.com/15kcNRb.png" group-title="", TVMÁS
+https://5ca9af4645e15.streamlock.net/rtv/videortv/chunklist_w2116443559.m3u8
```

Comment:
Link do not work.

Reply:
Updated link:
```suggestion
#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Spanish" tvg-logo="https://i.imgur.com/15kcNRb.png" group-title="", TVMÁS
https://5ca9af4645e15.streamlock.net/rtv/videortv/chunklist_w192674072.m3u8
```
The other day i saw that those three links stop working by a day or two, and then resume working again.